### ISSUE TITLE
Function Overloading

### ISSUE THREAD

**Issue by jeff-bradley** [CrunchyData/pg_featureserv#160]
is it possible to overload a function and have the same name but with different parameters? I am able to create both functions and pg_featureserv reads them both but when I check the functions in the UI they both accept the same parameters and when it gets called it only uses the function without the additional parameter.

**Comment by dr-jts** [comment]
I don't think this would be easy to implement, since it would require disambiguation logic in the server logic (but happy to be shown wrong).
